### ISSUE TITLE
Allow multiple loss functions

### ISSUE THREAD

**Issue by kaminow** [choderalab/asapdiscovery#1044]
We've tossed around ideas for training that include incorporating additional loss terms on top of the pure target value loss, eg penalizing for predictions outside a normal pIC50 range, additional cross entropy loss to ensure the best predicted pose is the closes to the crystal structure, etc.
